lib/output/writer/cache.go

```diff
@@ -87,6 +94,15 @@ func (c *Cache) WriteWithContext(ctx context.Context, msg types.Message) error {
 // Write attempts to write message contents to a target Cache.
 func (c *Cache) Write(msg types.Message) error {
 	if msg.Len() == 1 {
+		var ttl *time.Duration
+		var err error
+		if ttls := c.ttl.String(0, msg); ttls != "" {
+			*ttl, err = time.ParseDuration(ttls)
```

Comment:
I think this is an invalid pointer, it's also not used below.

Reply:
Oh, I forgot this one, I think it's done now.